core/lib/server/pubFields.ts

```diff
@@ -35,8 +35,7 @@ type GetPubFieldsInput =
  * Get pub fields
  *
  * @param props  - When nothing is supplied, return all the pub fields
- * @param props.pubId - When supplied, return all the pub fields associated with the pub through pub values and the pub type of that pub
- * When props.valuesOnly is true, only return the pub fields associated with the pub through pub values, not through the pub type
```

Comment:
the `valuesOnly` prop was removed from the type at some point, but the query was never updated. and now `valuesOnly` is actually the behavior we always want so i've updated the comments and removed the additional pub type fields join from the query.

i also checked and the only place we use this version of the query (passing the pubid) is on the pub editor that i'm fixing here.